flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/common/ViewsExpandingTest.scala

```diff
@@ -91,6 +91,32 @@ class ViewsExpandingTest(tableTestUtil: TableTestBase => TableTestUtil) extends
     tableUtil.verifyPlan(query)
   }
 
+  @Test
+  def testViewExpandingWithMismatchRowType(): Unit = {
+    val tableUtil = tableTestUtil(this)
+    val tableEnv = tableUtil.tableEnv
+    tableUtil.addDataStream[(Int, String, Int)]("t1", 'a, 'b, 'c)
+    def createAggSqlView(originTable: String): CatalogView = {
```

Comment:
nit: We don't need this method. We could just create the view. I will fix that while merging.